client/packages/invoices/src/Returns/modals/OutboundReturn/ReturnSteps.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  useTranslation,
+  WizardStepper,
+  TabPanel,
+  TabContext,
+  OutboundReturnLineNode,
+  RecordPatch,
+} from '@openmsupply-client/common';
+import { QuantityToReturnTable } from './ReturnQuantitiesTable';
+import { ReturnReasonsTable } from '../ReturnReasonsTable';
+
+export enum Tabs {
+  Quantity = 'Quantity',
+  Reason = 'Reason',
+}
+
+interface ReturnStepsProps {
+  currentTab: string;
+  lines: OutboundReturnLineNode[];
+  update: (patch: RecordPatch<OutboundReturnLineNode>) => void;
+}
+
+export const ReturnSteps = ({
+  currentTab,
+  lines,
+  update,
+}: ReturnStepsProps) => {
+  const t = useTranslation('replenishment');
+
+  const returnsSteps = [
+    { tab: Tabs.Quantity, label: t('label.select-quantity'), description: '' },
+    { tab: Tabs.Reason, label: t('label.reason'), description: '' },
+  ];
+
+  const getActiveStep = () => {
+    const step = returnsSteps.findIndex(step => step.tab === currentTab);
+    return step === -1 ? 0 : step;
+  };
+
+  return (
+    <TabContext value={currentTab}>
+      <WizardStepper activeStep={getActiveStep()} steps={returnsSteps} />
+      <TabPanel value={Tabs.Quantity}>
+        <QuantityToReturnTable
+          lines={lines}
+          updateLine={line => update(line)}
+        />
+      </TabPanel>
+      <TabPanel value={Tabs.Reason}>
+        <ReturnReasonsTable
+          lines={lines.filter(l => l.numberOfPacksToReturn > 0)}
```

Comment:
Filtering here so only need to add reasons/comments to the lines we're actually returning... does the UX feel alright to you?

Reply:
Can we disable/hide the "Okay & Next" button until some values are added? I agree, it's weird going to the second step and just seeing "Nothing here". 

I think the preferred UX is to keep the button active, but show an alert when trying to do something not allowed -- for example when you try and issue a quantity of 0 on an outbound shipment line. Maybe we should do that (can do in another issue if you want)

Reply:
Except the plan was to delete return lines by setting the return quantity to 0, so it's technically a valid input? I think I'll put it aside as something to discuss w PO?

Reply:
PO? 

I think my preferred solution would be to show a warning when *all* quantities are 0 and you click OK: "You have specified no quantities for any of the available lines. If you proceed then no lines will be added to the return", or something like that.

But yeah, something to discuss as a group.

Reply:
Ah haha as in Richard the PO 😁 Oo yeah a warning like that sounds like a good idea! Just might need to have a different message based on whether we are adding or editing - will add issue :)